### PR TITLE
Improve outputs of bin/generate-page script

### PR DESCRIPTION
**Pull request by paulbalandan**
- Adds colors to terminal output
- Splits words with spaces (e.g. SimplifiedChinese => Simplified Chinese)

### CODE REVIEW

File: bin/generate-page [hunk 1]
```diff
@@ -87,10 +89,18 @@ foreach ($locales as $localeKey => $locale) {
...
 
     $items[$locale] = [
-        'language' => str_replace(['Translations\\Tests\\', 'TranslationTest'], '', $localeKey),
-        'locale'   => $locale,
-        'percent'  => 0,
-        'files'    => [],
+        'language' => preg_replace_callback(
+            '/^Translations\\\\Tests\\\\(\S+)TranslationTest$/',
+            static fn (array $matches): string => preg_replace(
+                '/(?<!^)((?=[\p{Lu}][^\p{Lu}])|(?<![\p{Lu}])(?=[\p{Lu}]))/u',
```
Comment (MGatner) [inline bin/generate-page:95]: 😳 next level regexing

**Review by MGatner — APPROVED** [review]
Looks good! Please let @michalsn review.

**Review by michalsn — APPROVED** [review]
Looks great!

File: bin/generate-page [hunk 2]
```diff
@@ -87,10 +89,18 @@ foreach ($locales as $localeKey => $locale) {
     );
 
     $items[$locale] = [
-        'language' => str_replace(['Translations\\Tests\\', 'TranslationTest'], '', $localeKey),
-        'locale'   => $locale,
-        'percent'  => 0,
-        'files'    => [],
+        'language' => preg_replace_callback(
+            '/^Translations\\\\Tests\\\\(\S+)TranslationTest$/',
```
Comment (kenjis) [inline bin/generate-page:93]: I think using `\A` and `\z` is better than `^` and `$`.

**Pull request merged**